tree/ntuple/src/RMiniFile.cxx

```diff
@@ -714,46 +714,10 @@ std::uint64_t ROOT::Internal::RMiniFileReader::SearchInDirectory(std::uint64_t &
    return 0;
 }
 
-ROOT::RResult<ROOT::RNTuple> ROOT::Internal::RMiniFileReader::GetNTupleProper(std::string_view ntuplePath)
+ROOT::RResult<ROOT::RNTuple> ROOT::Internal::RMiniFileReader::GetNTupleProperAtSeekPos(std::uint64_t keySeekPos)
```

Comment:
Maybe move the new function after `GetNTupleProper`, as in the header? (this will also reduce the diff significantly because the existing function is just split)